lib/aws/ses.rb

```diff
@@ -23,24 +23,11 @@ def ses_client
       end
 
       def ses_client_options
-        opts = {
+        {
           # https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/timeout-duration.html
           retry_limit: 3,
           retry_backoff: ->(_context) { sleep(2) },
         }
-        opts[:region] = pick_region_from_pool if Figaro.env.aws_ses_region_pool.present?
```

Comment:
Should double check that the AWS SDK defaults to the right region when no value is set. Many AWS SDK libraries will error if the region isn't set by `AWS_REGION` or in config.

Reply:
We can look, I think it should. We actually did not set the region before we added the region pool.

Reply:
Just hopped on a prod idp and tried it out. Looks like we're good.